### PR TITLE
Use `transition` shorthand if appropriate

### PR DESCRIPTION
**Pull request by davidchambers**
This change addresses [#585](https://github.com/chriseppstein/compass/issues/585) in the manner that @paulirish suggested.

---

I've had quite a bit of difficulty running the test suite. Initially I made these changes on `stable` and was able to run the tests successfully (after patching one failing test). I then switched to `master`, applied the changes, and tried to run the tests again without success.

There's probably something wrong with my setup. When I run `rake run_tests` I get:

```
WARNING: no such file to load -- rcov/rcovtask
rake aborted!
```

On `stable` I was able to run the tests via `rake test`, but when I try that now on `master` I get `WARNING: DSL method Object#ruby` ad infinitum.

---

To summarize, I _think_ the patch is good, but I'd greatly appreciate some suggestions as to what I'm doing wrong so that I can confirm this to be the case.

### CODE REVIEW

**Comment by scottdavis** [comment]
<pre>
WARNING: no such file to load -- rcov/rcovtask
rake aborted!
</pre>

this is normal

also make sure to run `bundle exec rake test features` before committing and make sure you are green

You also need to create a test for this test files for css live in test/fixtures/stylesheets/compass/(css & sass)

you create the expected css output in the one folder and the sass required to generate it in the other it does compair white space so be careful

**Comment by davidchambers** [comment]
Thanks for explaining how to run the tests, Scott.

> You also need to create a test for this

Already done. :)

I'll run `bundle exec rake test features` when I get home and update the pull request if necessary.

**Comment by scottdavis** [comment]
sorry i read this request before i had my coffee let me know if everything passes and ill pull it in

**Comment by davidchambers** [comment]
Forgive me, Scott, but I'm new to this (exciting) world of rubies and gems.

This is what happened:

```
> bundle exec rake test features
/Users/dc/.rbenv/versions/1.9.2-p290/lib/ruby/gems/1.9.1/gems/bundler-1.0.21/lib/bundler/rubygems_integration.rb:219:in `block in replace_bin_path': can't find executable rake (Gem::Exception)
    from /Users/dc/.rbenv/versions/1.9.2-p290/lib/ruby/gems/1.9.1/bin/rake:19:in `<main>'
```

**Comment by scottdavis** [comment]
you may need to run just `bundle` before running that command or you may need to `gem install rake`

**Comment by davidchambers** [comment]
I just ran the tests on my work computer—I'm still not sure what's wrong with my setup at home—and all the tests passed. This is safe to pull.

**Comment by scottdavis** [comment]
word ill get this pushed in

**Comment by scottdavis** [comment]
BAM
